components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -245,8 +256,6 @@ lnd/clm2/surfdata_map/surfdata_ne4pg2_simyr2000_c190620.nc</fsurdat>
 lnd/clm2/surfdata_map/surfdata_ne240np4_simyr2000_c170821.nc</fsurdat>
 <fsurdat hgrid="r05"         sim_year="2000" >
 lnd/clm2/surfdata_map/surfdata_0.5x0.5_simyr2000_c190418.nc</fsurdat>
-<fsurdat hgrid="r0125"       sim_year="2000" use_crop=".false." >
-lnd/clm2/surfdata_map/surfdata_0.125x0.125_simyr2000_c190730.nc</fsurdat>
 
```

Comment:
@tangq, Why is this entry being removed?

Reply:
@bishtgautam , looks like this entry was removed by you in commit 631d2064f61d9ce37895ed152d2c85eb1809b7ee on Feb 14.

Reply:
I will look at the commit to figure out what I was doing.

Reply:
@bishtgautam , it works well this way. I also added one for r05 & simyr2010 via PR #3655 using this format.

Reply:
Is the intent here to not support `sim_year="2000"` and instead support `sim_year="2010"`?

Reply:
It is to ensure that compset F2010SC5-CMIP6 uses sim_year 2010 setting. Not an intent here to discontinue support for sim_year 2000.  We could keep the settings for both 2000 and 2010 in namelist_defaults_clm4_5.xml.

Reply:
Ok, I now understand the issue here.